@ecomplus/storefront-components/src/js/RecommendedItems.js

```diff
@@ -60,6 +60,12 @@ export default {
       default () {
         return ecomCart
       }
+    },
+    items: {
+      type: Array,
+      default() {
+        return []
+      }
```

Comment:
na verdade talvez seja melhor o `RecommendedItems` receber em prop o `productIds` em vez de `items`, aí ele mesmo faz o fetch apenas quando for iniciado, faz sentido?

> aí manteria `items` no data dele em vez de `localItems`, e teria `localProductIds` ou coisa parecida no data

Reply:
aqui eu só não entendi por que precisaria do localProductIds 🤔

Reply:
na verdade vi sua implementação e nem precisa dele mesmo, você fez melhor do que eu sugeri 👏🏾